spark/src/main/scala/ai/chronon/spark/ChrononKryoRegistrator.scala

```diff
@@ -127,7 +126,6 @@ class ChrononKryoRegistrator extends KryoRegistrator {
       "scala.reflect.ManifestFactory$LongManifest",
       "org.apache.spark.sql.execution.joins.EmptyHashedRelation$",
       "scala.reflect.ManifestFactory$$anon$1",
-      "scala.reflect.ClassTag$GenericClassTag",
```

Comment:
This might cause issues based on version of scala we are testing with. We recently disabled scala 2.11 tests - which is probably the version that removing this will break.

Let's remove this line.